### PR TITLE
feat(StatusListItem): scroll feature added if list of tags exceeds component width

### PR DESCRIPTION
**Pull request by saledjenic**

### CODE REVIEW

**Review by borismelnik — APPROVED** [review]

**Review by caybro — APPROVED** [review]
LGTM

Just the PR description is a bit lacking :)

**Comment by saledjenic** [comment]
> Just the PR description is a bit lacking :)

The change is so simple and title is descriptive enough, I had nothing different/more to add then what I've said in the title. :)

Thanks for the review.

File: sandbox/controls/ListItems.qml [hunk 1]
```diff
@@ -342,16 +342,37 @@ CExPynn1gWf9bx498P7/nzPcxEzGExhBdJGYihtAYQlO+tUZvqrPbqeudo5iJGEJjCE15a3VtodH3q2I
     StatusListItem {
         title: "List Item with Inline Tags"
         icon.isLetterIdenticon: true
-        tagsModel : 2
+        tagsModel: ListModel{
```
Comment (micieslak) [inline sandbox/controls/ListItems.qml:345]: minor: missing space before {

**Review by micieslak — APPROVED** [review]
lgtm, but having description would be cool ;)

**Pull request merged**